next.config.js

```diff
@@ -0,0 +1,58 @@
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  async redirects() {
+    return [
+      {
+        source: "/docs",
+        destination: "/docs/guide/introduction",
+        permanent: false,
+      },
+      {
+        source: "/docs/guide",
+        destination: "/docs/guide/introduction",
+        permanent: false,
+      },
+      {
+        source: "/docs/components",
+        destination: "/docs/components/overview",
+        permanent: false,
+      },
+      {
+        source: "/templates",
+        destination: "/templates/minimal/overview",
+        permanent: false,
+      },
+      {
+        source: "/templates/minimal",
+        destination: "/templates/minimal/overview",
+        permanent: false,
+      },
+    ];
+  },
```

Comment:
We have a lot of redirects here, I think we could remove a significant part of them if we could move the `Overview` pages from `/docs/guide/overview` to just `/docs/guide` as we tried at one point. So the final structure would look like this:

![image](https://github.com/webscopeio/mailingui/assets/48859243/27eb6d27-8607-4936-8c77-b2760683eb54)

I know we had some issues with this, and I don't know how much time you spend on making this work so if you feel like getting this to work is not worth it let me know 😆.

Reply:
Let's give it a shot https://github.com/webscopeio/mailingui/issues/216